src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx

```diff
@@ -239,6 +239,19 @@ export default function NurseryWithdrawalsDetails({
             </TabPanel>
           </TabContext>
         )}
+        {withdrawal?.purpose === OUTPLANT && !withdrawalSummary?.hasReassignments && (
+          <Box sx={contentPanelProps}>
```

Comment:
we should probably title this section as 'Details' instead of 'Withdrawal' to stay consistent with the other views that don't have tabs?
Also.. seems like a disabled Reassigment tab would be most helpful here instead of hiding the tab - which would confuse the users. But.. that would be a PM decision.

Reply:
Thanks Karthik. @rachel-tf-design what are your thoughts here?

Reply:
Looks good as in, you're happy with it as is? Or Karthik's suggestions look good? Or?

Reply:
Looks good as in I'm happy with it as is. I'd prefer not disabling the reassignments tab because we don't expect reassignment to happen much nor are we really encouraging it, so to me, it makes more sense to have it hidden until it's necessary to show.

Reply:
👍 alright. Thank you for the clarification!

Reply:
> Looks good as in I'm happy with it as is. I'd prefer not disabling the reassignments tab because we don't expect reassignment to happen much nor are we really encouraging it, so to me, it makes more sense to have it hidden until it's necessary to show.
If we aren't encouraging it - the dedicated reassign column in the withdrawal log sends a mixed message.

Reply:
Yeah, the design and product team discussed this a while back and debated where to put that action (i.e. reassign). Since we're not encouraging it, we didn't want to make it a primary page level button but also, since it's the only real action a user can take on that page, we didn't want to hide it in a kebab icon. As a result, we settled on make it a secondary row level button.

Reply:
> Yeah, the design and product team discussed this a while back and debated where to put that action (i.e. reassign). Since we're not encouraging it, we didn't want to make it a primary page level button but also, since it's the only real action a user can take on that page, we didn't want to hide it in a kebab icon. As a result, we settled on make it a secondary row level button.

Thanks @rachel-tf-design this context helps. I should read the design notes more often, my bad.

Reply:
No worries! This discussion was during the designing stage, so before we did the first handoff. I'll try to do a better job of documenting context/reasoning for design decisions in the future :)